.travis/linux.win64.install.sh

```diff
@@ -3,10 +3,12 @@ set -e
 
 # First, install 32-bit deps
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-. $DIR/linux.win32.install.sh
+# Maybe find a better fix: see https://github.com/koalaman/shellcheck/wiki/SC1090
+# shellcheck disable=SC1090
```

Comment:
I'll do some ugly debugging using Travis here. I'll remove the two lines anyway once it's done. Just a reminder to myself. :)